src/ripple/app/ledger/tests/common_ledger.h

```diff
@@ -236,11 +240,23 @@ trust(TestAccount& from, TestAccount const& issuer,
                 std::string const& currency, double amount,
                 Ledger::pointer const& ledger, bool sign = true);
 
+// Simplest form of close_and_advance.
 void
 close_and_advance(Ledger::pointer& ledger, std::shared_ptr<Ledger const>& LCL);
 
-Json::Value findPath(Ledger::pointer ledger, TestAccount const& src, 
-    TestAccount const& dest, std::vector<Currency> srcCurrencies, 
+// Call this function if you want to advance the ledger time.
+void
+close_and_advance(Ledger::pointer& ledger,
+    std::shared_ptr<Ledger const>& LCL, std::uint32_t advSeconds);
+
+// Call this function if you want to retry transactions that fail with
+// a `ter` or `tec`.
+void
+close_and_advance(Ledger::pointer& ledger, std::shared_ptr<Ledger const>& LCL,
+    std::uint32_t advSeconds, std::unique_ptr<LocalTxs>& retries);
```

Comment:
Functionality for advancing the ledger and dealing with retries should be in jtx. And we should not use `LocalTxs` to store the list of retriables.